dj/movies/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
```

Comment:
Remove `rest_framework` we don't require this. I have already sent you a template please use that. `django.http.JsonResponse` will work.